### PR TITLE
DOCS-2998: TC 2.1 update alert manage article 

### PR DESCRIPTION
**Pull request by micjohnson777**
Updates the alertManage.md article with 2.1 changes to screens, functions and procedures. Reorganizes content into tabs
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/TrueCommand/Alerts/AlertManage.md [hunk 1]
```diff
@@ -6,125 +6,179 @@ aliases:
...
-## Viewing All Alerts
+TrueCommand provides three alert screen options:
+* **All Alerts** which displays all system alerts discovered by TrueCommand on the systems it monitors.
+* **Alert Rules** which allows adminstrators and users with permissions to configure alerts for monitored systems.
+* **Alert Services** which allows adminstrators to configure communication plugins.
 
-To see all alerts that TrueCommand has discovered, open the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and click **All Alerts**.
-Administrator accounts can see all generated alerts.
-A non-administrator account can only view alerts according to their team and user account permissions.
+Alerts inidications display on the **All Alerts**, **Dashboard** and **Systems** screens.
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:17]: typo
```suggestion
Alerts indications display on the **All Alerts**, **Dashboard** and **Systems** screens. 
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 2]
```diff
@@ -6,125 +6,179 @@ aliases:
...
+![DashboardSystemAlerts](/images/TrueCommand/2.0/DashboardSystemAlerts.png "Dashboard System Alerts")
 
-Each active and resolved alert for that system is visible in this tab.
-Clicking *View alert notice details (`i`)* shows details for that alert, including the option to leave comments about the alert.
+To view all alerts for a single system, from the **Dashboard** and click on a system name with alerts indicated. That system's details screen displays with the alerts listed in the **Alerts** area of the screen.
 
-## Managing Alert Rules
+![DashboardSystemDetailAlerts](/images/TrueCommand/2.1/DashboardSystemDetailAlerts.png "Dashboard System Detail Alerts")
+
+Click the <mat-icon _ngcontent-bbg-c138="" role="img" fontset="mdi" fonticon="mdi-comment-text-multiple" class="mat-icon notranslate mdi mdi-comment-text-multiple mat-icon-no-color" aria-hidden="true" data-mat-icon-type="font" data-mat-icon-name="mdi-comment-text-multiple" data-mat-icon-namespace="mdi"></mat-icon> icon in the **Alerts** area of the system detail screen to display the **Notice Details** side panel with information about the alert and any user comments made.
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:40]: This <mat-icon> code should be able to be pared down to the <span class="iconify" data-icon="mdi:comment-text-multiple"></span>
```suggestion
Click the <span class="iconify" data-icon="mdi:mdi-comment-text-multiple"></span> icon in the **Alerts** area of the system detail screen to display the **Notice Details** side panel with information about the alert and any user comments made.
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 3]
```diff
@@ -6,125 +6,179 @@ aliases:
...
+
+To view all alerts from the **Systems** screen system list, click the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and then click **Systems**.
+
+![SystemsList](/images/TrueCommand/2.1/SystemsList.png "Systems List")
+
+Just as with the systems detail alerts, click on the <mat-icon _ngcontent-bbg-c138="" role="img" fontset="mdi" fonticon="mdi-comment-text-multiple" class="mat-icon notranslate mdi mdi-comment-text-multiple mat-icon-no-color" aria-hidden="true" data-mat-icon-type="font" data-mat-icon-name="mdi-comment-text-multiple" data-mat-icon-namespace="mdi"></mat-icon> icon in the **COMMENTS** column to display the **Notice Details** side panel with information about the alert and any user comments made.
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:46]: As above, pared down icon call:
```suggestion
Just as with the systems detail alerts, click on the <span class="iconify" data-icon="mdi:comment-text-multiple"></span> icon in the **COMMENTS** column to display the **Notice Details** side panel with information about the alert and any user comments made.
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 4]
```diff
@@ -6,125 +6,179 @@ aliases:
...
+
+## Resolving Alerts
+
+You can move an alert in the **All Alerts** screen to the **RESOLVED** column by clicking the **Resolve** <i class="material-icons" aria-hidden="true" title="Check">check</i> to the right of the delete icon.
+
+To resolve multiple alerts, select each alert checkbox and then click **RESOLVE SELECTED**.
+
+Click the <mat-icon _ngcontent-bbg-c138="" role="img" fontset="mdi" fonticon="mdi-comment-text-multiple" class="mat-icon notranslate mdi mdi-comment-text-multiple mat-icon-no-color" aria-hidden="true" data-mat-icon-type="font" data-mat-icon-name="mdi-comment-text-multiple" data-mat-icon-namespace="mdi"></mat-icon> icon in the **COMMENTS** column to display the **Notice Details** side panel with information about the alert and any user comments made.
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:54]:
```suggestion
Click the <span class="iconify" data-icon="mdi:comment-text-multiple"></span> icon in the **COMMENTS** column to display the **Notice Details** side panel with information about the alert and any user comments made.
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 5]
```diff
@@ -6,125 +6,179 @@ aliases:
...
+
+![AllAlertDetails](/images/TrueCommand/2.1/AllAlertDetails.png "All Alert Details")
+
+To resolve an alert on the system detail screen **Alerts** area, click on the **Resolve alert notice** <i class="material-icons" aria-hidden="true" title="Check">check</i> to the left of the comments icon.
+
+![SystemResolveAlertNotice](/images/TrueCommand/2.1/SystemResolveAlertNotice.png "System Resolve Alert Notice")
+
+## Deleting Alerts
+
+Administrator accounts can delete an alert by clicking the delete icon <i class="material-icons" aria-hidden="true" title="Delete">delete</i>.
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:64]: Made consistent with previous usage.
```suggestion
Administrator accounts can delete an alert by clicking the <i class="material-icons" aria-hidden="true" title="Delete">delete</i> icon.
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 6]
```diff
@@ -6,125 +6,179 @@ aliases:
...
+* **SMTP Email** is a plugin to configure system and user email services.
 
-<mat-icon _ngcontent-ath-c575="" role="img" fontset="mdi" fonticon="mdi-test-tube" class="mat-icon notranslate mdi mdi-test-tube mat-icon-no-color" aria-hidden="true" ng-reflect-font-set="mdi" ng-reflect-font-icon="mdi-test-tube"></mat-icon> : Sent Test Alert
+![Alert Services Screen](/images/TrueCommand/2.1/AlertServiceScreen.png "Alert Services")
 
-<i class="material-icons" aria-hidden="true" title="Settings">settings</i> : Configure Plugin
+Each plugin has three options:
 
-<mat-icon _ngcontent-ath-c200="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">remove_circle</mat-icon> : Clear Plugin Configuration
+* **Send test email** <mat-icon _ngcontent-ath-c575="" role="img" fontset="mdi" fonticon="mdi-test-tube" class="mat-icon notranslate mdi mdi-test-tube mat-icon-no-color" aria-hidden="true" ng-reflect-font-set="mdi" ng-reflect-font-icon="mdi-test-tube"></mat-icon>
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:129]:
```suggestion
* **Send test email** <span class="iconify" data-icon="mdi:test-tube"></span>
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 7]
```diff
@@ -6,125 +6,179 @@ aliases:
...
+![Alert Services Screen](/images/TrueCommand/2.1/AlertServiceScreen.png "Alert Services")
 
-<i class="material-icons" aria-hidden="true" title="Settings">settings</i> : Configure Plugin
+Each plugin has three options:
 
-<mat-icon _ngcontent-ath-c200="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">remove_circle</mat-icon> : Clear Plugin Configuration
+* **Send test email** <mat-icon _ngcontent-ath-c575="" role="img" fontset="mdi" fonticon="mdi-test-tube" class="mat-icon notranslate mdi mdi-test-tube mat-icon-no-color" aria-hidden="true" ng-reflect-font-set="mdi" ng-reflect-font-icon="mdi-test-tube"></mat-icon> 
+* **Configure plugin** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> 
+* **Clear plugin configuration** <mat-icon _ngcontent-ath-c200="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">remove_circle</mat-icon>
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:131]: Is this one working on builds?
```suggestion
* **Clear plugin configuration** <mat-icon _ngcontent-ath-c200="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">remove_circle</mat-icon> 
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 8]
```diff
@@ -6,125 +6,179 @@ aliases:
...
+   
+   ![AlertServicesSMTPEmailptions](/images/TrueCommand/2.0/AlertServicesSMTPEmailptions.png "Alert Services: SMTP Email options")
 
-Click **Test** to verify that the configuarion is correct.
-If you did not receive a test alert email, check the values entered for accuracy.
+   * **Mailserver** for example *smtp.gmail.com*
+   * **Mailserver port** number
+   * **Auth user** email address for plain authentication, for example, *adminuser@yourmail.com*
+   * **Auth pass** password for the plain authentication; for a *No-Auth* SMTP configuration, leave the password field blank
+   * **From** is the what sends the eamil (i.e., no-reply@TrueCommand.io),or allows you to customize the sender field of the email
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:151]: typos
```suggestion
   * **From** is what sends the email (i.e., no-reply@TrueCommand.io), or allows you to customize the sender field of the email 
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 9]
```diff
@@ -6,125 +6,179 @@ aliases:
...
-If you did not receive a test alert email, check the values entered for accuracy.
+   * **Mailserver** for example *smtp.gmail.com*
+   * **Mailserver port** number
+   * **Auth user** email address for plain authentication, for example, *adminuser@yourmail.com*
+   * **Auth pass** password for the plain authentication; for a *No-Auth* SMTP configuration, leave the password field blank
+   * **From** is the what sends the eamil (i.e., no-reply@TrueCommand.io),or allows you to customize the sender field of the email 
 
-{{< /tab >}}
-{{< tab "PagerDuty" >}}
+  Click **Test** on the **SMTP Email** configuration screen to verify that the configuarion is correct.
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:153]: typo
```suggestion
  Click **Test** on the **SMTP Email** configuration screen to verify that the configuration is correct.
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 10]
```diff
@@ -6,125 +6,179 @@ aliases:
...
-Enter a *Title for incident reports* if desired. Click *TEST*.
+   The system should automatically send a test email to the specified email address but if not click **VERIFY EMAIL**.  
+   A **Confirmation** dialog box displays with a field where you type or copy/paste the code sent in the email. Doing this verifies the email.
+   
+   ![SMTPSetupVerifyEmail](/images/TrueCommand/2.1/SMTPSetupVerifyEmail.png "SMTP Setup Verify Email")
+
+## Configuring PagerDuty
+
+Open the **Configure Plugin** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> for PagerDuty. In the
+**Authtoken** section, enter your **Service Integration Key from PagerDuty**.  If you have an active subscription with PagerDuty, this key should be available to you.
```
Comment (Mrt134) [inline content/TrueCommand/Alerts/AlertManage.md:171]: Kind of an odd line break. Simple suggestion to break on the sentence.
```suggestion
Open the **Configure Plugin** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> for PagerDuty.
In the **Authtoken** section, enter your **Service Integration Key from PagerDuty**. If you have an active subscription with PagerDuty, this key should be available to you. 
```

**Review by Mrt134 — COMMENTED** [review]
Found a couple typos and some spots where pared-down icon code could be helpful

File: content/TrueCommand/Alerts/AlertManage.md (hunk 10 above)
Comment (micjohnson777) [reply content/TrueCommand/Alerts/AlertManage.md:171]: That was an oops... I didn't catch it. Thanks

File: content/TrueCommand/Alerts/AlertManage.md (hunk 7 above)
Comment (micjohnson777) [reply content/TrueCommand/Alerts/AlertManage.md:131]: This was the original icon link content that worked I just moved it to display after the 'label' for it for consistent parallel construction in the bullet list. 
My local build worked, but the hint box I added after it was messed up so I removed the hint box and it worked without issues.

**Review by BonnieF — CHANGES_REQUESTED** [review]
content/SCALE/Storage/Pools/PermissionsSCALE.md line 54 should read "Confirm changes to the user." and line 56 should read "Confirm changes to Group." since the User and Group are present tense in this list.

Line 85 should read "The full pathway to the file." to keep it consistent with the other two descriptions in that table.

Line 118 also should read "The full pathway to the file." to keep it consistent with the other two descriptions in that table.

Line 129 should read "Add a new ACE to the Access Control List."

Line 136 Should start off with "Removed all ACL's . . ."

and maybe line 137, the first sentence, might say something like "Load a preset ACL configured to match general permissions situations.


On content/SCALE/SystemSettings/AdvancedSettings.md

Line 58 might start with  "Activate a tuning script . . . . . ."

**Pull request merged**